sql.tf

```diff
@@ -56,7 +57,7 @@ resource "aws_db_instance" "default" {
   allocated_storage     = var.sql_instance_allocated_storage
   max_allocated_storage = var.sql_instance_max_allocated_storage
 
-  storage_type   = "gp2"
+  storage_type   = "gp3"
```

Comment:
Create a variable: `var.sql_storage_type` and set the default to gp2

Reply:
fixed